src/domain/categories.ts

```diff
@@ -45,46 +42,81 @@ export function editCategory(props: Category) {
 
 //ini namanya custom hooks
 export function useFetchCategories() {
-  const [categories, setCategories] = useState<Category[]>();
+  const [categories, setCategories] = useState<Category[]>([]);
+  const [errorMessageFetch, seterrorMessageFetch] = useState<string>("");
```

Comment:
Ini bisa pake camel case

```suggestion
  const [errorMessageFetch, setErrorMessageFetch] = useState<string>("");
```

Reply:
Done Update